CBLServerWrapper/CBLServerWrapper/ServerManager.cs

```diff
@@ -40,7 +48,8 @@ public static void StartServer(string fileName, MainWindow mainWindow)
             MinecraftServer.Exited += MinecraftServer_Exited;
             ServerJarPath = fileName;
             ConsoleWindow = mainWindow;
-            mainWindow.Title = "Minecraft Server Wrapper - " + MinecraftServer.StartInfo.WorkingDirectory.Split('\\').Last();
+            GetWorldName(MinecraftServer.StartInfo.WorkingDirectory);
+            mainWindow.Title = MinecraftServer.StartInfo.WorkingDirectory.Split('\\').Last() + " | " + WorldName;
```

Comment:
This will print "{Folder Name} | {World Name}" as the window title.
You probably wanted something like "Minecraft Server Wrapper | {World Name}"